app/data_mapper_setup.rb

```diff
@@ -0,0 +1,10 @@
+env = ENV['RACK_ENV'] || "development"
+
+DataMapper.setup(:default, "postgres://localhost/bookmark_manager_#{env}")
+
+require_relative './lib/tweet'
+require_relative './lib/user'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+DataMapper.auto_migrate!
```

Comment:
we don't want to do the upgrade or the migrate in this file but in the Rakefile.